src/esp/bindings/SensorBindings.cpp

```diff
@@ -156,6 +157,7 @@ void initSensorBindings(py::module& m) {
   py::class_<SensorSuite, SensorSuite::ptr>(m, "SensorSuite")
```

Comment:
Seems like this never got removed, it should have been.  Nothing uses this -- it's just a dictionary so python just uses a `dict`